screen2.0/src/common/components/mainsearch/SnpAutocomplete.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from "react"
+import Box from "@mui/material/Box"
+import TextField from "@mui/material/TextField"
+import Autocomplete from "@mui/material/Autocomplete"
+import Grid from "@mui/material/Grid"
+import Typography from "@mui/material/Typography"
+import { debounce } from "@mui/material/utils"
+import { useRouter } from "next/navigation"
+
+const SNP_AUTOCOMPLETE_QUERY = `
```

Comment:
Like above comment, maybe put in queries file or leave as is if it's not used elsewhere